### DISCUSSION TITLE
Opt-in to Hacktoberfest

### DISCUSSION THREAD

**Discussion by SjorsO** [laravel/framework#34709] [General] [answered]
This year, repositories have to opt-in to [Hacktoberfest](https://hacktoberfest.digitalocean.com/) in order for PRs to count towards the 4 PR goal.

It would be nice if the Laravel repositories opted-in so contributors can get rewarded with some stickers and a t-shirt. [This post](https://hacktoberfest.digitalocean.com/hacktoberfest-update) explains in detail how to opt-in (it's easy, just add the "hacktoberfest" tag to each repository).

**Comment by driesvints** [comment, marked as answer]
We aren't going to opt in to hacktoberfest due to the influx of spam we've already received.

**Reply by SjorsO** [reply]
Understandable. 

Do you mind manually tagging PRs with `hacktoberfest-accepted` if the contributor asks for it? (given it isn't a low quality PR, of course)

**Reply by driesvints** [reply]
No, we don't want to maintain any labels related to a specific event.
